src/planning/plan/tests/test-environment-test.ts

```diff
@@ -7,17 +7,20 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-import {registerSystemExceptionHandler} from '../../../runtime/arc-exceptions.js';
+import {registerSystemExceptionHandler, removeSystemExceptionHandler, defaultSystemExceptionHandler} from '../../../runtime/arc-exceptions.js';
 
-let exceptions = [];
+let exceptions: Error[] = [];
 
-beforeEach(() => registerSystemExceptionHandler((exception) => exceptions.push(exception)));
+before(() => {
+  removeSystemExceptionHandler(defaultSystemExceptionHandler);
+  registerSystemExceptionHandler((exception) => exceptions.push(exception));
+});
 
 afterEach(function() {
   if (exceptions.length > 0) {
-    for (const exception of exceptions) {
-      this.test.ctx.currentTest.err = exception;
-    }
+    const exception = exceptions[0];
```

Comment:
Is it worth running through all of them?

Reply:
We mostly want the test to report as failed; adding extra exceptions after the first doesn't really make the failure cause any clearer.